cli/azd/cmd/env.go

```diff
@@ -493,3 +491,19 @@ func (eg *envGetValuesAction) Run(ctx context.Context) (*actions.ActionResult, e
 
 	return nil, nil
 }
+
+func getCmdEnvHelpDescription(*cobra.Command) string {
+	return generateCmdHelpDescription(
+		"Manage your application environments. With this command group, you can create a new environment or get, set,"+
+			" and list your application environments.",
+		[]string{
+			formatHelpNote("An Application can have multiple environments (ex: dev, test, prod)."),
+			formatHelpNote("Each environment may have a different configuration (that is, connectivity information)" +
+				" for accessing Azure resources."),
+			formatHelpNote(fmt.Sprintf("You can find all environment configuration under the %s folder.",
+				output.WithLinkFormat(".azure/<environment-name>"))),
+			formatHelpNote(fmt.Sprintf("The environment name is stored as the %s environment variable in the %s file.",
+				output.WithHighLightFormat("AZURE_ENV_NAME"),
+				output.WithLinkFormat(".azure/<environment-name>/.env"))),
+		})
+}
```

Comment:
We don't use `cobra.Command` at all here. Do you think it's feasible to avoid `generator` functions and simply return plain text?

Reply:
> Do you think it's feasible to avoid generator functions and simply return plain text?

isn't `string` == plain text?  or do you mean `[]byte` ?   

> We don't use cobra.Command

The cobra command is passed to generators in case they want to use it, but using it is not a `must`